unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/TextureHelpers/TextureHelpers.cs

```diff
@@ -76,4 +77,17 @@ public static Texture2D CopyTexture(Texture2D sourceTexture)
         
         return texture;
     }
+
+    private const string TEXTURE_COMPRESSION_FLAG_NAME = "tex_compression";
+#if UNITY_STANDALONE
+        bool compressTextures = false
+#else
+    private static bool compressTextures = DataStore.i.featureFlags.flags.Get().IsFeatureEnabled(TEXTURE_COMPRESSION_FLAG_NAME);
```

Comment:
How does this work with race conditions? Because at the moment of initialization of the static bool, feature flags are not available. Thus, it may use a value that is not the one that comes from the feature flag system. It may work a better % of times for query params, but definitely not for the ffsystem.

A solution would be to always ask for the fresh `DataStore.i.featureFlags.flags.Get().IsFeatureEnabled(TEXTURE_COMPRESSION_FLAG_NAME)` in the Compress function instead of using the static bool

Reply:
Yes, I just didn't want to ask `IsFeatureEnabled()` all the time but I guess I'll do something like that then

Reply:
done, changed approach based on Alex feedback.